src/github.ts

```diff
@@ -136,7 +136,7 @@ export const createGithubRelease = async (
   appOctokit: Octokit,
   owner: string,
   repo: string,
-  productionRelease: boolean,
+  productionDelivery: string,
```

Comment:
🔧 Why did you change this to string? This should be a boolean.

Reply:
This what integreat is using at the moment.